workflow-bot-app/src/main/java/com/symphony/bdk/workflow/configuration/WorkflowDataSourceConfiguration.java

```diff
@@ -35,4 +38,12 @@ public PlatformTransactionManager transactionManager(@Qualifier("datasource") Da
     return transactionManager;
   }
 
+  @Bean
+  @ConditionalOnProperty(value = "wdk.workflows.path", havingValue = "not-empty")
```

Comment:
I think there is a default for wdk.workflow.path, shouldn't this be executed in this case too?

Reply:
The default value won't impact this check.